Bylaws.tex

```diff
@@ -99,11 +103,11 @@ \part{Active Organization}
 
 \article{Judiciary Committee}
 
-\section A Judiciary Committee shall be formed in the event that a dispute occurs that cannot be handled by the Vice-President alone. The Vice-President in this case will nominate three non-partisan brothers to be approved by the Executive Council.
+\section A Judiciary Committee shall be formed in the event that a dispute occurs that cannot be handled by the Vice-President alone or in an instance where Standards Board cannot resolve the dispute due to a conflict of interest, as determined by the Vice-President. The Vice-President in this case will nominate three non-partisan brothers to be approved by the Executive Council.
```

Comment:
The active chapter should come back to this in the future to add a clause that accounts for the case that no set of three non-partisan brothers exist to handle this issue. It is so approved.